internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -120,3 +121,17 @@ func (e *EnvStackConfig) ToEnv(stack *cloudformation.Stack) (*archer.Environment
 
 	return &createdEnv, nil
 }
+
+// EnvStack is the cloudformation stack created for an environment.
+type EnvStack struct {
+	*cloudformation.Stack
+}
+
+// ExecutionRoleARN returns the execution role ARN from the stack.
+func (s *EnvStack) ExecutionRoleARN() string {
```

Comment:
Maybe we could throw this into ToEnv ? That way you'd get the env manager role and execution role.

Reply:
Done!